src/elements/alt-style/AltStyleElementForm.tsx

```diff
@@ -3,27 +3,32 @@ import { neutral } from "@guardian/src-foundations";
 import React from "react";
 import { FieldLayoutVertical } from "../../editorial-source-components/FieldLayout";
 import { FieldWrapper } from "../../editorial-source-components/FieldWrapper";
+import type { RepeaterFieldDescription } from "../../plugin/fieldViews/RepeaterFieldView";
 import { RepeaterFieldMapIDKey } from "../../plugin/helpers/constants";
+import type {
+  FieldDescriptions,
+  FieldNameToField,
+} from "../../plugin/types/Element";
 import { AltStyleElementWrapper } from "../../renderers/react/AltStyleElementWrapper";
 import { createReactElementSpec } from "../../renderers/react/createReactElementSpec";
 import { Body } from "../../renderers/react/ElementWrapper";
 import {
   LeftRepeaterActionControls,
   RightRepeaterActionControls,
 } from "../../renderers/react/WrapperControls";
-import { altStyleFields } from "./AltStyleElementSpec";
+import { keyTakeawaysFields } from "./AltStyleElementSpec";
 
 export const AltStyleElementTestId = "AltStyleElement";
 
-const RepeaterBody = styled(Body)`
+const RepeaterChild = styled(Body)`
   padding: 8px 8px 16px 8px;
   &:not(:last-child) {
     border-bottom: 1px dashed ${neutral[60]};
   }
   position: relative;
 `;
 
-const RepeaterChild = styled("div")`
+const FieldsWrapper = styled("div")`
```

Comment:
perhaps... 
```suggestion
const RepeatedFieldsWrapper = styled("div")`
```
...would be better?